src/calib_feat.bpf.c

```diff
@@ -33,7 +33,13 @@ int calib_entry(struct pt_regs *ctx)
 	/* Used for kretprobe function entry IP discovery, before
 	 * bpf_get_func_ip() helper was added.
 	 */
-	entry_ip = PT_REGS_IP(ctx) - 1;
+	entry_ip = PT_REGS_IP(ctx);
```

Comment:
this is very subjective, I realize, but I find it simpler to reason about stuff like this if we have #if/#else doing full expressions (and also #ifdef seems cleaner in this case?):

```
#ifdef bpf_target_x86
    entry_ip = PT_REGS_IP(ctx) - 1;
#else
    entry_ip = PT_REGS_IP(ctx);
#endif
```

Of course keep the comment :)